javascript/code-challenges/tree/tree.js

```diff
@@ -53,8 +53,43 @@ class BinaryTree {
     return newArray;
   }
 
+  //////////////////////////////////////////////////////
+  // code challenge 16
+  findMaximumValue() {
+    let arrayValues = inOrder(root);
+
+    for (i = 0; i <= arrayValues.length; i++) {
+      let max = 0;
+
+      if (arrayValues[i] > max)
+        max = arrayValues[i];
+    }
+
+    return max;
+  }
+  ///////////////////////////////////////////////////////
+
+  breadthFirst(root) {
+    if (!root) throw new Error;
+    let array = [];
+    let currentNode = this.root;
+    array.push(currentNode.value);
```

Comment:
It looks like there is an issue with the `.value` coming back as undefined.
When you have a chance to come back to this and get the test passing I will regrade for you!